config/samples/nextsim.dg.yml

```diff
@@ -7,7 +7,9 @@ job_submit:
   scheduler: oar
   ppn: 32
   project: pr-sasip
+  # one can choose `devel` queue or `standard` queue
   queue: devel
+  walltime: 1800
```

Comment:
In your current system, `walltime` is only used here and the one defined in `model_def` section is never used I think.

Reply:
we shall use the walltime defined in model_def to model.run jobs. Because for different models maybe the walltime shall be different, and it shall also be different from the walltime for enkf program.

In models/nextsim/dg/model.py run_batch function, the run_job command shall take additional "walltime=self.walltime" to make the one defined in model_def work

Reply:
will change this in a follow-up commit in next pull request